django-workload/django_workload/inbox.py

```diff
@@ -199,7 +205,7 @@ def post_process(self, result):
 
 class InboxConfig(object):
     def __init__(self):
-        self.mult_factor = 1000
+        self.mult_factor = 700
```

Comment:
Can we add a comment as to what `mult_factor` and `load_mult` exactly do for people who don't know the code as well please.

Reply:
Please see my latest commit. Thanks for the feedback :)